extensions/amp-story/0.1/amp-story-page.js

```diff
@@ -288,24 +288,20 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @public
    */
   calculateLoadStatus() {
-    if (this.isLoaded_ || this.pageElements_.length == 0) {
+    const visiblePageElements = this.pageElements_.filter(pageElement =>
+      !pageElement.isHiddenByMediaQuery());
```

Comment:
Minor: Move this invocation after `this.isLoaded_` check to avoid wasted work.

```js
if (this.isLoaded_) {
  return true;
}

const visiblePageElements = ...
if (visiblePageElements.length == 0) {
  return true;
}
```

Reply:
True, much better.